docs/network/evm-nodes.md

```diff
@@ -43,3 +90,8 @@ The default is `$HOME/.octez-evm-node`.
 
 By default, the EVM node exposes its JSON RPC API endpoint to `localhost:8545`.
```

Comment:
I can't write a thread for the line 88. It's only a suggestion to inform users that `--initial-kernel` is only needed for the first run, after that it can be ommitted.

Reply:
Added "The `--initial-kernel` argument is needed only the first time that you start the node."

Reply:
Whoops, this change missed the PR. Adding it here: https://github.com/etherlinkcom/docs/pull/88